Bio/PDB/Mutate.py

```diff
@@ -0,0 +1,578 @@
+    },
+    "CHI4": {
+        "ARG": {"axis": ["CD", "NE"], "ref_plane": ["CG", "CD", "NE", "CZ"]},
+        "LYS": {"axis": ["CG", "CE"], "ref_plane": ["CG", "CD", "CE", "NZ"]},
+    },
+}
+
+RESIDUE_ORDER = {
+    "CYS": ["N", "CA", "C", "O", "CB", "SG"],
+    "ASP": ["N", "CA", "C", "O", "CB", "CG", "OD1", "OD2"],
+    "SER": ["N", "CA", "C", "O", "CB", "OG"],
+    "GLN": ["N", "CA", "C", "O", "CB", "CG", "CD", "NE2", "OE1"],
+    "LYS": ["N", "CA", "C", "O", "CB", "CG", "CD", "CE", "NZ"],
+    "ILE": ["N", "CA", "C", "O", "CB", "CG1", "CG2", "CD1"],
+    "PRO": ["N", "CA", "C", "O", "CB", "CG", "CD"],
+    "THR": ["N", "CA", "C", "O", "CB", "CG2", "OG1"],
+    "PHE": ["N", "CA", "C", "O", "CB", "CG", "CD1", "CD2", "CE1", "CE2", "CZ"],
+    "ASN": ["N", "CA", "C", "O", "CB", "CG", "ND2", "OD1"],
+    "GLY": ["N", "CA", "C", "O"],
+    "HIS": ["N", "CA", "C", "O", "CB", "CG", "CD2", "ND1", "CE1", "NE2"],
+    "LEU": ["N", "CA", "C", "O", "CB", "CG", "CD1", "CD2"],
+    "ARG": ["N", "CA", "C", "O", "CB", "CG", "CD", "NE", "CZ", "NH1", "NH2"],
+    "TRP": [
+        "N",
+        "CA",
+        "C",
+        "O",
+        "CB",
+        "CG",
+        "CD1",
+        "CD2",
+        "CE2",
+        "CE3",
+        "NE1",
+        "CZ2",
+        "CZ3",
+        "CH2",
+    ],
+    "ALA": ["N", "CA", "C", "O", "CB"],
+    "VAL": ["N", "CA", "C", "O", "CB", "CG1", "CG2"],
+    "GLU": ["N", "CA", "C", "O", "CB", "CG", "CD", "OE1", "OE2"],
+    "TYR": ["N", "CA", "C", "O", "CB", "CG", "CD1", "CD2", "CE1", "CE2", "CZ", "OH"],
+    "MET": ["N", "CA", "C", "O", "CB", "CG", "SD", "CE"],
+}
+
+
+def load_rotamers(rotamer_loc=DUNBRACK_FILTERED):
+    """
+    Load the Dunbrack rotamer library
+    """
+    _dunbrack = {}
+    with gzip.open(rotamer_loc, "rt") as fn:
+        for line in fn:
+            if line.startswith("#"):
+                continue
+            if line.split()[0] not in _dunbrack:
+                _dunbrack[line.split()[0]] = {}
+            if int(line.split()[1]) not in _dunbrack[line.split()[0]]:
+                _dunbrack[line.split()[0]][int(line.split()[1])] = {}
+            if (
+                int(line.split()[2])
+                not in _dunbrack[line.split()[0]][int(line.split()[1])]
+            ):
+                _dunbrack[line.split()[0]][int(line.split()[1])][
+                    int(line.split()[2])
+                ] = []
+            _dunbrack[line.split()[0]][int(line.split()[1])][
+                int(line.split()[2])
+            ].append(
+                {
+                    "prob": float(line.split()[3]),
+                    "CHI1": float(line.split()[4]),
+                    "CHI2": float(line.split()[5]),
+                    "CHI3": float(line.split()[6]),
+                    "CHI4": float(line.split()[7]),
+                }
+            )
+    return _dunbrack
+
+
+def rotation_matrix(axis, theta):
+    """
+    Calculates a rotational matrix alongside an axis with an andle theta
```

Comment:
_andle_ -> _angle_
and I think:
_alongside_ -> _about_ or _around_

Reply:
This has been fixed!